package.json

```diff
@@ -85,7 +85,7 @@
     "lru-cache": "^11.0.0",
     "masto": "^6.7.5",
     "node-emoji": "^2.1.3",
-    "nuxt-security": "^1.4.3",
+    "nuxt-security": "^2.0.0",
```

Comment:
It seems that this major version up somehow broke the type inference of the returned value of `useRuntimeConfig()`.

Reply:
OK, the latest version of `nuxt-security` (v2.1.4) no longer has the previous type error.